client/my-sites/navigation/index.jsx

```diff
@@ -21,7 +20,17 @@ class MySitesNavigation extends React.Component {
 	};
 
 	render() {
-		const SidebarComponent = config.isEnabled( 'jetpack-cloud' ) ? JetpackCloudSidebar : Sidebar;
+		const asyncProps = {
+			placeholder: null,
+			path: this.props.path,
+			siteBasePath: this.props.siteBasePath,
+		};
+
+		const asyncSidebar = config.isEnabled( 'jetpack-cloud' ) ? (
+			<AsyncLoad require="landing/jetpack-cloud/components/sidebar" { ...asyncProps } />
```

Comment:
Could we pass the `require` to `asyncProps` and define it up there too? This way we won't have to write `<AsyncLoad ... />` twice, and the ternary will be used only to determine the differences without any duplication.